.gitignore

```diff
@@ -1,3 +1,24 @@
 /.idea/
 *.iml
-/out/
```

Comment:
Please make sure don't write all exclusions one by one use generic regex

Reply:
deleted all the production files and added only *production to git ignore